Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_VertexData.azsli

```diff
@@ -23,6 +23,15 @@
 #define VsOutput        VsOutput_BasePBR
 #endif
 
+#ifndef ENABLE_VERTEX_COLOR
+#define ENABLE_VERTEX_COLOR 1
+#endif
```

Comment:
I'm wondering if 'enabled' is the correct default. Currently, shaders that use it (BasePBR, StandardPBR and EnhancedPBR) add vertex color and other code, and shaders that don't (skin, multilayer) need to specify:
#define ENABLE_VERTEX_COLOR 0
Would it be better if the default is 0, and shaders that use it add both
#define ENABLE_VERTEX_COLOR 0
and the vertex color attribute in their vertex data, and materials that don't use it don't need to do anything at all? Specifying that you want to enable the thing you're adding seems less error prone that disabling the thing you're not adding.
Thoughts?